src/Enum/LabelSizeEnum.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Enum;
+
+class LabelSizeEnum
+{
+    public const SIZE_A4 = 'A4';
+
+    public const SIZE_A5 = 'A5';
+
+    public const SIZE_A6 = 'A6';
```

Comment:
I believe for "etykiety" very common size is A6P (at least "kurierskie"), so I propose to add support for that size.

Please consider.

Reply:
I checked and A6P has the same dimensions as A6 size